modules/account-creation/main.tf

```diff
@@ -1,15 +1,9 @@
-provider "aws" {
-  region = "us-east-1"
-}
-
 # this must be a unique email address
 # this will be used to perform the root user password reset when cleaning up (deleting) the account
 # (or use something like a gmail alias, ex: <username>+aws.orgs.dev@gmail.com)

```

Comment:
I like the use of the gmail dot blindness!

Reply:
All @jameshounshell idea.